client/src/parser/convertedModelForTesting.json

```diff
@@ -0,0 +1,351 @@
+{
```

Comment:
Does this file really belong into the main branch? In my opinion it should be excluded from merging

Reply:
We actually don't need it. I just deleted it.